packages/secure_storage/amplify_secure_storage/android/src/main/kotlin/com/amazonaws/amplify/amplify_secure_storage/EncryptedKeyValueRepository.kt

```diff
@@ -63,7 +63,11 @@ open class EncryptedKeyValueRepository(
 
     @VisibleForTesting
     open fun removeSharedPreferencesFile() {
-        File("${context.filesDir.parent}/shared_prefs/$sharedPreferencesName.xml").delete()
+        val file = File("${context.filesDir.parent}/shared_prefs/$sharedPreferencesName.xml")
```

Comment:
Amplify Android handles it a bit differently. We will discuss aligning separately from this.

I think we need to use the File API to overwrite the contents in order to prevent this from surfacing as a potential issue in static analysis scans.